Resources/Prototypes/_Goobstation/Balloons/ai_faction.yml

```
@@ -0,0 +1,17 @@
+# SPDX-FileCopyrightText: 2025 Arraydeess <arraygitbash@gmail.com>
+# SPDX-FileCopyrightText: 2025 GoobBot <uristmchands@proton.me>
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+
+# A custom AI faction 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)